tests/wasmfs/wasmfs_seek.c

```diff
@@ -31,8 +14,9 @@
 #include <sys/uio.h>
 #include <unistd.h>
 
-// FIXME: Merge this standalone test back into dup.c after new FS can support
-// it.
+// FIXME: The existing io.c file implements devices, which haven't been
+// implemented in the new FS yet.
+// This test simply implements the seek portion.
```

Comment:
As we discussed this morning, it would be good to delete the relevant portions out of io.c (and maybe point to this file as its replacement).